packages/grid/_modules_/grid/hooks/features/core/gridState.ts

```diff
@@ -66,7 +69,9 @@ export const getInitialGridState: () => GridState = () => ({
   scrollBar: { hasScrollX: false, hasScrollY: false, scrollBarSize: { x: 0, y: 0 } },
   viewportSizes: { width: 0, height: 1 },
   sorting: getInitialGridSortingState(),
-  keyboard: { cell: null, columnHeader: null, isMultipleKeyPressed: false },
+  keyboard: { isMultipleKeyPressed: false },
+  focus: { cell: null, columnHeader: null },
+  tabIndex: { cell: null, columnHeader: null },
```

Comment:
This looks better. We should soon be able to kill `keyboard` with #1437 from @m4theushw.
I would also advocate to kill `focus` but it will be for another discussion.